packages/atomic/src/index.html

```diff
@@ -82,7 +82,7 @@
             </div>
 
             <div class="row">
-              <atomic-result-list>
+              <atomic-result-list fields-to-include='["language"]'>
```

Comment:
If you know it works, you can remove the attribute here, as language is already a default field.